global_config/find-domain.sh

```diff
@@ -0,0 +1,12 @@
+CONFIG=production
+CONFIG_SHORT=pd
+ENV_SHORT=${CONFIG_SHORT}
+AZ_SUBSCRIPTION=s189-teacher-services-cloud-production
+AZURE_SUBSCRIPTION=${AZ_SUBSCRIPTION}
+RESOURCE_PREFIX=s189p01
+ENV_TAG=Prod
+RESOURCE_GROUP_NAME=${RESOURCE_PREFIX}-${DNS_ZONE}domains-rg
+KEYVAULT_NAME=${RESOURCE_PREFIX}-${DNS_ZONE}domains-kv
+STORAGE_ACCOUNT_NAME=${RESOURCE_PREFIX}${DNS_ZONE}domainstf
+DOMAINS_ID=find
+DNS_ZONE=ftt
```

Comment:
RESOURCE_GROUP_NAME, KEYVAULT_NAME and STORAGE_ACCOUNT_NAME depend on DNS_ZONE. It should be moved before them.
Similary for `publish-domain.sh`